proto/osmosis/concentrated-liquidity/pool-model/tx.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package osmosis.concentratedliquidity.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/model";
+
+service MsgCreator {
+  rpc CreateConcentratedPool(MsgCreateConcentratedPool)
+      returns (MsgCreateConcentratedPoolResponse);
+}
+
+// ===================== MsgCreateConcentratedPool
+message MsgCreateConcentratedPool {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  string denom0 = 2 [ (gogoproto.moretags) = "yaml:\"denom0\"" ];
+  string denom1 = 3 [ (gogoproto.moretags) = "yaml:\"denom1\"" ];
+  uint64 tick_spacing = 4 [ (gogoproto.moretags) = "yaml:\"tick_spacing\"" ];
+}
```

Comment:
Oh interesting, yeah I guess there is no initial full range liquidity for CL pools

Reply:
Swap fee planned for a future PR?

Reply:
Yes!